include/prte.h

```diff
@@ -21,6 +21,7 @@
 /**
  * Main body of prte functionality
  */
-int prte(int argc, char *argv[]);
```

Comment:
i would suggest leaving this prototype in the header file for now in case something is using it.

Reply:
Hi, just for clarification would the header file include both prte and prte_launch then for the time being? 
for example:

/**
 * Main body of prte functionality
 */
int prte(int argc, char *argv[]);
int prte_launch(int argc, char *argv[]);

Reply:
Yes.

Reply:
yes keep both prototypes in the header file, ```prte``` and  ```prte_launch```